From 0bfc0d48ef7beadaf1c757b35ad6661c94cdecdb Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 4 Jul 2021 13:48:03 +0000
Subject: [PATCH] GCC: fix brace-initialization with NoDestructor

GCC requires brace-initialization for NoDestructor and std::vector<T>
for kSupportedFeatures. Therefore change to double brace-initialzation.
---
 device/vr/orientation/orientation_device.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/device/vr/orientation/orientation_device.cc b/device/vr/orientation/orientation_device.cc
index dbc90ae..684ba02 100644
--- a/device/vr/orientation/orientation_device.cc
+++ b/device/vr/orientation/orientation_device.cc
@@ -41,11 +41,11 @@ display::Display::Rotation GetRotation() {
 
 const std::vector<mojom::XRSessionFeature>& GetSupportedFeatures() {
   static base::NoDestructor<std::vector<mojom::XRSessionFeature>>
-      kSupportedFeatures({
+      kSupportedFeatures{{
     mojom::XRSessionFeature::REF_SPACE_VIEWER,
     mojom::XRSessionFeature::REF_SPACE_LOCAL,
     mojom::XRSessionFeature::REF_SPACE_LOCAL_FLOOR,
-  });
+  }};
 
   return *kSupportedFeatures;
 }
-- 
2.31.1

